scss/navbar.scss

```diff
@@ -0,0 +1,30 @@
+@import 'variables';
+
+.navbar {
+  position: fixed;
```

Comment:
When you add a fixed or absolute position you should add two position properties, you are using `top`, add one for horizontal position, like `left` or `right`